Gemfile

```diff
@@ -62,6 +62,6 @@ group :development do
   gem 'web-console', '>= 3.3.0'
   # Spring speeds up development by keeping your application running in the background.
   # Read more: https://github.com/rails/spring
-  gem 'spring'
-  gem 'spring-watcher-listen', '~> 2.0.0'
```

Comment:
@dmitry-sinina why did you remove spring? Have you encountered some issues with it?